### DISCUSSION TITLE
`Manage supported culture` should _imply_ `Manage Settings`

### DISCUSSION THREAD

**Discussion by jean** [OrchardCMS/OrchardCore#15849] [Q&A]
It is possible to assign `Manage Settings` and `Manage supported culture` independently. 
If `Manage supported culture` is assigned, the link to _manage supported culture_ will show up in the admin interface; however trying to access it will fail (because `Manage Settings`is not assigned).

A permission in Orchard can _imply_ other permissions.
Should `Manage supported culture` _imply_ `Manage Settings`?

What are the UI implications for this? When you assign a permission, all implied permission should be listed and/or notified to the user.

**Comment by hishamco** [comment]
> Should Manage supported culture imply Manage Settings?

I think it should be, because managing supported cultures is a part of manage setting

> When you assign a permission, all implied permission should be listed and/or notified to the user.

Agree

**Comment by hishamco** [comment]
> I think it should be, because managing supported cultures is a part of manage setting

The problem will face here if we allow this one, the user will be able to change the "General Settings" which is totally wrong. I may check this one

**Comment by hishamco** [comment]
IMHO `ManageSettings` should allow user to manage general settings, other than that all sub menu items underneath should restricted by the permission that defined in the `AdminMenu`

/cc @sebastienros @matiasmolleja
